participant/serializer.py

```diff
@@ -205,7 +227,26 @@ class ParticipantScoreSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Participant
-        fields = ('id', 'scores')
+        fields = ("id", "scores")
+
+    def get_scores(self, participant):
+        return participant.scores
+
+
+class ParticipantScoreSerializer(serializers.ModelSerializer):
```

Comment:
There is already another serializer with the same name. You would have to eliminate the previous one and keep this one, which is where you have added the new changes.